### ISSUE TITLE
Support slasher_active in clientstats

### ISSUE THREAD

**Issue by kasey** [prysmaticlabs/prysm#8850]
# 🚀 Feature Request

### Description
The client-stats spec specifies a metric in the `beaconnode` message type that tracks whether the validator has been activated. Currently the slasher is implemented as a separate process, so we don't have a good way to track this. But once https://github.com/prysmaticlabs/prysm/issues/8331 is merged, we will be able to add this to our payload.

 A clear and concise description of the problem or missing capability...

### Describe the solution you'd like
Once validator has been merged into beacon node, expose a prometheus metric to represent whether the slasher is active. Scrape this metric and use it to create the appropriate value for `slasher_active` in the client-stats payload.

 If you have a solution in mind, please describe it.
:point_up: 

### Describe alternatives you've considered

 Have you considered any alternative solutions or workarounds?
